internal/httpserve/handlers/checkout.go

```diff
@@ -0,0 +1,76 @@
+package handlers
+
+import (
+	"github.com/rs/zerolog/log"
+	"github.com/stripe/stripe-go/v81"
+	echo "github.com/theopenlane/echox"
+
+	"github.com/theopenlane/iam/auth"
+
+	"github.com/theopenlane/core/pkg/models"
+)
+
+// CheckoutSessionHandler is responsible for handling requests to /checkout/session and creating a stripe checkout session based on the user's organization context
+func (h *Handler) CheckoutSessionHandler(ctx echo.Context) error {
+	reqCtx := ctx.Request().Context()
+
+	orgID, err := auth.GetOrganizationIDFromContext(reqCtx)
+	if err != nil {
+		log.Error().Err(err).Msg("unable to get organization id from context")
+
+		return h.BadRequest(ctx, err)
+	}
+
+	log.Warn().Msg("obtained organization ID from context")
+
+	settings, err := h.getOrgSettingByOrgID(reqCtx, orgID)
+	if err != nil {
+		log.Error().Err(err).Msg("unable to get organization settings by org id")
+
+		return h.BadRequest(ctx, err)
+	}
+
+	log.Warn().Msg("obtained organization settings by org ID")
```

Comment:
Same on on these `Warns` below...
```suggestion
	log.Debug().Msg("obtained organization settings by org ID")
```